keras_cv/layers/feature_pyramid.py

```diff
@@ -137,6 +147,19 @@ def __init__(
         # the same merge layer is used for all levels
         self.merge_op = tf.keras.layers.Add()
 
+        if self.num_extra_level:
+            self.extra_level_layers = {}
+            # Note that the extra levels starts with self.max_level + 1
+            for i in range(self.num_extra_level):
+                level = i + self.max_level + 1
+                self.extra_level_layers[level] = tf.keras.layers.Conv2D(
+                    self.num_channels,
+                    kernel_size=3,
```

Comment:
from RetinaNet, this seems to be the case. Should we allow configurable parameters for kernels size, conv type, etc??

Reply:
So far I didn't see any customization for this case, we could expose more params when we hit those use cases. Note that the `num_extra_level` is only used by the retina-net at the moment.